llvm/test/tools/repo2obj/extern_mixed_weak_global_references.json

```diff
@@ -21,31 +21,29 @@
     {
       // generation 1
```

Comment:
Should change  `generation` to `transaction`?

Reply:
As above